extensions/amp-script/0.1/amp-script.js

```diff
@@ -25,18 +31,33 @@ import {
 import {dev, user} from '../../../src/log';
 import {getMode} from '../../../src/mode';
 import {isExperimentOn} from '../../../src/experiments';
-import {
-  sanitizer,
-  upgrade,
-} from '@ampproject/worker-dom/dist/unminified.index.safe.mjs.patched';
 
 /** @const {string} */
 const TAG = 'amp-script';
 
 /** @const {number} */
 const MAX_SCRIPT_SIZE = 150000;
 
+const PHASE_MUTATING = 2;
```

Comment:
Export from worker-dom?

Reply:
Currently can't re-export `Phase` from worker-dom - it inlines all constants and doesn't exist as an entity.